dlrover/python/tests/test_elastic_training_agent.py

```diff
@@ -363,7 +364,7 @@ def test_sync_node_port(self):
             log_dir=self.config.log_dir,
         )
         agent.sync_training_ports()
-        self.assertEqual(os.environ["HCCL_IF_BASE_PORT"], "60000")
+        self.assertEqual(os.environ[AscendConstants.HCCL_PORT_START], "64000")
```

Comment:
reuse the same constant as comments above.

Reply:
done